state_synchronizer/src/coordinator.rs

```diff
@@ -142,8 +145,10 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
                                     }
                                     if message.has_chunk_response() {
                                     if let Err(err) = self.process_chunk_response(message.take_chunk_response()).await {
+                                            self.peer_manager.update_score(&peer_id, PeerScoreUpdateType::InvalidChunk);
```

Comment:
we should be more careful here and double check that all potential errors in `process_chunk_response` are indeed can happened only because of peer malicious activity